pkg/handler/handler.go

```diff
@@ -191,9 +191,9 @@ func NewHandler(b broker.Broker, brokerConfig *config.Config, prefix string,
 		createVarHandler(h.lastoperation)).Methods("GET")
 
 	if brokerConfig.GetBool("broker.dev_broker") {
-		h.router.HandleFunc("/apb/spec", createVarHandler(h.apbAddSpec)).Methods("POST")
-		h.router.HandleFunc("/apb/spec/{spec_id}", createVarHandler(h.apbRemoveSpec)).Methods("DELETE")
-		h.router.HandleFunc("/apb/spec", createVarHandler(h.apbRemoveSpecs)).Methods("DELETE")
+		s.HandleFunc("/v2/apb", createVarHandler(h.apbAddSpec)).Methods("POST")
+		s.HandleFunc("/v2/apb/{spec_id}", createVarHandler(h.apbRemoveSpec)).Methods("DELETE")
+		s.HandleFunc("/v2/apb", createVarHandler(h.apbRemoveSpecs)).Methods("DELETE")
```

Comment:
are we intentionally droping the `/spec` and including this in `.../v2/...` now?

Reply:
I was thinking we could do that going forward.  Do you think we should keep it as is?

Reply:
no real opinion either way. This will need follow up PRs for the APB tool I think if we do make the change. Other than that it LGTM